### ISSUE TITLE
No menubar icon on Big Sur

### ISSUE THREAD

**Issue by fharper** [kitze/twizzle#43]
There is no icon in the menubar on Big Sur which make it impossible to tweet with the app.

**Comment by kitze** [comment]
I fixed this but I cannot build the project now, yay for super old dependencies!

**Comment by jbenton** [comment]
Just wanted to say Twizzle has been absolutely ESSENTIAL for me and my productivity the past few years, so if you need any user love to update this, here it is! :) It worked fine for me in Big Sur for a long time, but it just randomly disappeared from my menu bar a week or two ago and I haven't been able to figure out any way to get it back.

**Comment by kitze** [comment]
@jbenton yeah it pisses me off too but I didn't have time to fix it :( thank you btw!
